src/aiogram_dialog/utils.py

```diff
@@ -174,3 +176,18 @@ def remove_indent_id(callback_data: str) -> Tuple[Optional[str], str]:
         intent_id, new_data = callback_data.split(CB_SEP, maxsplit=1)
         return intent_id, new_data
     return None, callback_data
+
+
+def sync_scroll(scroll_id: str) -> Callable[
```

Comment:
I guess we could support multiple ids here.

Additionally, as it is used as `on_page_changed` the user will loose ability set own callback, so we can support it here.